### ISSUE TITLE
Feature Request: Changeable DL Server

### ISSUE THREAD

**Issue by Sebi94nbg** [TS3Tools/TS3UpdateScript#64]
> Hello,
> 
> I love your script but I unfortunatly can't use it.
> 
> My dedicated server hosting company's IPs (online.net) has been blocked 
> by 4netplayers,
> 
> so the script can't download the ts3 package from 4netplayers mirror.
> 
> Could you add a feature that allow to change the mirror to gamed!de ?
> 
> I tried to change the download link in the script, but there is a lot of 
> them !
> 
> A new feature would be really appreciated.
> 
> 
> Best Regards,
> 
> A. P.

**Comment by Sebi94nbg** [comment]
Due of that, that the web server of `gamed!de` just returns an 404 error code, I'm unable to see the structure and content of it. But it looks like, that it's similar to `4Netplayers.de`:
```
http://dl.4players.de/ts/releases/3.0.13.6/teamspeak3-server_linux_amd64-3.0.13.6.tar.bz2
http://teamspeak.gameserver.gamed.de/ts3/releases/3.0.13.6/teamspeak3-server_linux_amd64-3.0.13.6.tar.bz2

http://dl.4players.de/ts/releases/3.0.13.6/teamspeak3-server_freebsd_amd64-3.0.13.6.tar.bz2
http://teamspeak.gameserver.gamed.de/ts3/releases/3.0.13.6/teamspeak3-server_freebsd_amd64-3.0.13.6.tar.bz2
```
The only difference should be the hostname and instead of "ts", they're using "ts3".

Unfortunately, I need to find a solution for how to detect the latest stable/beta version using `gamed!de`. See issue #65.

**Comment by Sebi94nbg** [comment]
To test it, I've set a local `/etc/hosts` entry, which simply points to my loopback IP address and tried to update my test TS3 server.

Due of that, that the script tries to download the release first from 4Netplayers mirror and isn't able to download it, the mirror switches to gamed.

In my test case, the server was updated successful.

I've just sent a pre-release of the TS3UpdateScript to the client, who requested this feature to let him test it too, because his IP address is blocked on one of the mirrors.
